client/src/components/LogCard.jsx

```diff
@@ -7,45 +7,32 @@ import Air from './card/Air';
 import Water from './card/Water';
 import Buddy from './card/Buddy';
 
-const initialLog = {
-  id: 1,
-  date: '16.08.2021',
-  time: '13:48',
-  duration: '48',
-  country: 'Egypt',
-  spot: 'Abu Nashim',
-  airStart: '200',
-  airEnd: '60',
-  airEan: '32',
-  waterDepthMax: '28.4',
-  waterDepthAvg: '17.3',
-  waterTemp: '23',
-  notes:
-    'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Corporis dolor dignissimos, aliquam officiis itaque consequuntur eveniet quia soluta eius sint?',
-  buddy: 'Antonia',
-};
 
-function LogCard() {
+function LogCard({ data }) {
+
   return (
     <CardContainer>
-      <Title logData={initialLog} />
+      <Title logData={data} />
```

Comment:
In order to see which properties of the `data` object are needed and used by the components, it might be better to use specific props (e.g. `date={data.date}` )and only pass the needed props into the component